mlflow/protos/databricks_artifacts_pb2.py

```diff
@@ -24,7 +24,7 @@
   package='mlflow',
```

Comment:
Automatically regenerated to reflect changes in `mlflow/protos/databricks_artifacts.proto` (see https://github.com/mlflow/mlflow/pull/4402/files#r641035500).